prac_07/project.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+
+class Project:
+    def __init__(self, name, start_date, priority, cost_estimate, completion_percentage):
+        self.name = name
+        self.start_date = datetime.strptime(start_date, "%d/%m/%Y")
+        self.priority = int(priority)  # Ensure priority is stored as an integer
```

Comment:
You may want to add validation to ensure priority is within an expected range? I mean this is fine enough too.